src/TimeGridEvent.js

```diff
@@ -49,7 +49,7 @@ function TimeGridEvent(props) {
           top: stringifyPercent(top),
           [rtl ? 'right' : 'left']: stringifyPercent(xOffset),
           width: stringifyPercent(width),
-          height: stringifyPercent(height),
+          height: `calc(${stringifyPercent(height)} - 2px)`,
```

Comment:
1px should be good

Reply:
@manishsingh08 changed it to 1 px but the change is not noticeable, i just changed it back to 2px so the bubble appear above the grey line of the calendar